### PR TITLE
Set docker replicas and memory limit

### PR DESCRIPTION
**Pull request by palvarez89**
Replicas so that if a container is somehow killed, there is another one to serve requests.

Memory limits so that if a container starts consuming lots of ram, it gets killed.

### CODE REVIEW

**Comment by palvarez89** [comment]
RFC for now, wanna know your thoughts.

File: roles/echaloasuerte-3/tasks/main.yml [hunk 1]
```diff
@@ -121,6 +121,8 @@
       retries: 10
       start_period: 30s
     replicas: "{{ echaloasuerte_3_replicas }}"
+    limits:
+      memory: 300M
```
Comment (mariocj89) [inline roles/echaloasuerte-3/tasks/main.yml]: Do we know what is the baseline?

Comment (palvarez89) [reply roles/echaloasuerte-3/tasks/main.yml]: I've been checkin, and 180M is what I've seen normally. But this is a good point, I'll try to get some data and make some graphs

Comment (palvarez89) [reply roles/echaloasuerte-3/tasks/main.yml]: As promised

Screenshot from 2020-01-08 11-16-44

They consume between 200M and 250M, but I know they peak when initialising the containers. I'll get more data, and do some testing, to see what happens with temporary spikes.

Given this data I would suggest 350M-400M of memory limit, no more.

Comment (mariocj89) [reply roles/echaloasuerte-3/tasks/main.yml]: OK, my fear is if we get a really hot day with some raffles that take too much memory that this system might cause a continous reboot and cause more harm than good. But I trust your judgement.

Comment (mariocj89) [reply roles/echaloasuerte-3/tasks/main.yml]: Bump it to 400 and LGTM.

Comment (palvarez89) [reply roles/echaloasuerte-3/tasks/main.yml]: I will try to evaluate what happens in those cases, and continue monitoring before I move forward

**Pull request closed without merging**